src/DGtal/geometry/curves/FrechetShortcut.ih

```diff
@@ -830,7 +830,7 @@ bool DGtal::FrechetShortcut<TIterator,TInteger>::isBackpathOk()
   Point firstP = Point(*myBegin);
   Point P = Point(*(myEnd+1));
   
-  int q = Tools::computeQuadrant(firstP,P);
+  int q = Tools::computeOctant(firstP,P);
```

Comment:
why did you change this  ? I have the feeling that it would change the behavior of the method..